journal/src/test/java/io/zeebe/journal/file/JournalTest.java

```diff
@@ -426,7 +429,7 @@ void shouldReopenJournalWithExistingRecords() throws Exception {
   @Test
   void shouldReadReopenedJournal() throws Exception {
     // given
-    final var appendedRecord = journal.append(data);
+    final var appendedRecord = copyRecord(journal.append(data));
```

Comment:
:+1: It's weird that the test didn't fail always. :slightly_smiling_face:

Reply:
The new ones are slightly slower so I guess it's enough for the memory to get collected before the assertion